src/inscription.rs

```diff
@@ -189,6 +230,10 @@ impl Inscription {
     str::from_utf8(self.content_type.as_ref()?).ok()
   }
 
+  pub(crate) fn content_encoding(&self) -> Option<&str> {
```

Comment:
Return `Option<HeaderValue>`, and content encoding values which are invalid header values will be ignored.